### DISCUSSION TITLE
Setting Burn bundle variables.

### DISCUSSION THREAD

**Discussion by jeffu231** [wixtoolset/issues#7246] [Questions] [answered]
I am trying to use code like the following similar to many examples I have seen.

`<Variable Name="InstallFolder" bal:Overridable="yes" Type="string" Value="[ProgramFiles6432Folder]$(var.Manufacturer)\$(var.Name)"/>
`

However I find that the value of ProgramsFiles6432Folder is not being replaced. The preprocessor variables work fine. I was tooling around the schema docs and noticed an option for the Type as formatted so I tried this.

`<Variable Name="InstallFolder" bal:Overridable="yes" Type="formatted" Value="[ProgramFiles6432Folder]$(var.Manufacturer)\$(var.Name)"/>
`

Using the Type of formatted then all worked as expected. Is this a change in V4 or is there some functionality I am not understanding.

**Comment by robmen** [comment, marked as answer]
> Is this a change in V4

Yes. I thought there was a WIP, but a quick search only found https://github.com/wixtoolset/issues/issues/4763.

**Reply by jeffu231** [reply]
Thanks for the confirmation I am on the right path. Might be worth a mention in the new docs since that pattern is pretty prevalent.

**Reply by rseanhall** [reply]
The [WIP](https://wixtoolset.org/docs/development/wips/thmutil-literal-variables/) was tied to a different issue - https://github.com/wixtoolset/issues/issues/4863.

Documentation PRs are always welcome at https://github.com/wixtoolset/web.
